src/EStimLibrary/Core/HardwareInterfaces/Lead.cs

```diff
@@ -46,9 +46,10 @@ public bool GetConnectedOutputs(int id, bool searchIsAnOutput,
 
         var validId = false;
         // Search by output or contact ID, respectively.
-        if (searchIsAnOutput && (validId = this.OutputSet.Contains(id)))
+        if (searchIsAnOutput)
```

Comment:
Not relevant to this line, but I think you can delete the using statement at the top of this file. Also if you wouldn't mind adding doc comments for the class fields and properties (there are only 2 and one already has a comment, just change it to /// with a summary tag instead of a simple in-ilne //)